std/range/package.d

```diff
@@ -7130,6 +7130,17 @@ sgi.com/tech/stl/binary_search.html, binary_search).
         }
         return false;
     }
+
+// groupBy
+/**
+Returns a range of subranges of elements that are equivalent according to the
+sorting relation.
+ */
+    auto groupBy()()
```

Comment:
This is a template because chunkBy fails to compile for the SortedRange created by the sort call in makeIndex (the sort predicate references a local variable). I haven't been able to reduce yet, but this is a temporary workaround to ensure that the chunkBy is only instantiated when you try to use it.